deployer/template/aws_lambda_permission.go

```diff
@@ -0,0 +1,23 @@
+package template
+
+import (
+	"github.com/awslabs/goformation/cloudformation"
+	"github.com/awslabs/goformation/cloudformation/resources"
+)
+
+func ValidateAWSLambdaPermission(
+	projectName, configName, resourceName string,
+	template *cloudformation.Template,
+	res *resources.AWSLambdaPermission,
+) error {
+	if res.Action != "lambda:InvokeFunction" {
+		return resourceError(res, resourceName, "Lambda::Permission.Action must be lambda:InvokeFunction")
+	}
+
```

Comment:
Restriction on SourceArn being defined in the same template, i.e. a `!Ref` perhaps

Reply:
we actually don't set source arn at the moment, because it created a circular dependency with targetGroup -> lambda -> permission -> targetGroup